dev-docs/bidder-adaptor.md

```diff
@@ -271,6 +271,20 @@ Building the request will use data from several places:
 * **BidRequest Params**: Several important parameters such as first-party data, userId, GDPR, USP, and supply chain values are on the `bidderRequest` object.
 * **Prebid Config**: Publishers can set a number of config values that bid adapters should consider reading.
 
+#### Compression Support for Outgoing Requests
+
+Prebid.js core now includes support for gzip compression of bidder request payloads (This can help reduce payload size and improve performance).
+
+The following criteria must be met for a request to be compressed:
+
+* Browser supports gzip compression
+* Participating bidders have enabled compression support within their bid adapter by updating their outgoing bidder requests to include `request.options.endpointCompression = true`
+* Participating bidders have implemented compression support on their server-side endpoint
+
+If the above criteria is met, Prebid core will automatically compress outgoing data payloads.
```

Comment:
Prebid will pass the compressed payload if `request.options.endpointCompression = true` is set by bidder and browser is compatible

Reply:
we can even mention that PubMatic bidder already utilizes this feature, refer the code.